pkg/tests/cross-tests/defaults.go

```diff
@@ -0,0 +1,9 @@
+package crosstests
+
+var (
```

Comment:
I have trouble imagining that these will change.
```suggestion
const (
```